ClinGen/clincoded
# Connects to #395. Help link to Quick Start Guide pdf.

This PR is for adding a question mark icon button linking to the Quick Start Guide pdf.

**Steps to test:**
1. Login and expect to see the question mark link button appearing in the top navbar as well as on the Dashboard (next to "Select Variant for Variant Curation" link).
2. Click on either one of the buttons and expect to open a new tab/window with the Quick Start Guide pdf.
3. Log out and expect to see the question mark link button no longer available in the top navbar.
